canonical/ubuntu.com
# WD-2132 Update the content of the ROS ESM page content

## Done

- Updated the content of the ROS ESM page to match the [copy doc](https://docs.google.com/document/d/1oiz_006n9U1Ya50wdSkKIyj_jsxSr92FdCpkDU4kpHs/edit#)
- Created a pattern for the new tiered card pattern and applied it to both the "/robotics/ros-esm" and "/pro" pages

## QA

- Check the demo
- Go to /robotics/ros-esm page and check it matches the copy doc above
- Go to the /pro page and see that is the same
